projects/plugins/jetpack/require-lib.php

```diff
@@ -1,10 +1,6 @@
 <?php
 function jetpack_require_lib( $slug ) {
-	if ( !preg_match( '|^[a-z0-9/_.-]+$|i', $slug ) ) {
```

Comment:
Time sink. Not needed, as `require_once` will fatal if it doesn't find any of the files.